### DISCUSSION TITLE
Custom Code blocks

### DISCUSSION THREAD

**Discussion by Joly0** [requarks/wiki#6597] [Help / Questions] [answered]
Hey, i would like to be able to have more options for "Insert code block" shortcut. For example i would like to use sql or powershell, but these arent available as an option.

Somewhere else i read this should work:
````
```SQL
// some code here 
```
````

But that doesnt work sadly :(

**Comment by Joly0** [comment]
Ok, i have seen this only works in markdown editor, not in wysiwyg. Is there any solution to use this in this editor aswell? Or any option to add custom entries to the code blocks shortcut?

**Comment by NGPixel** [comment, marked as answer]
No. Use the markdown editor.

Support for the visual editor is coming in a future release.
